docs/source/delta-utility.md

```diff
@@ -82,6 +86,16 @@ default retention threshold for the files is 7 days. To change this behavior, se
       deltaTable.vacuum(100);     // vacuum files not required by versions more than 100 hours old
       ```
 
+#### Inventory Table
```

Comment:
can you add this as the last section under vacuum. Otherwise it may look like the notes and stuff are related to inventory table mode.